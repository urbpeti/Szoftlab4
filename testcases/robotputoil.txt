Robot puts down oil
Robot R1 35 2
Put R1 Oil
Step
List_Robots
List_Items