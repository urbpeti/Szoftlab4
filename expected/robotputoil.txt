R1, Position: 37.0, Velocity: 2.0, Not Oily, Alive
Oil 37.0 0
